public/options.js

```diff
@@ -32,53 +34,15 @@ document.getElementById("toggleEnabled").addEventListener("click", async event =
 });
 
 document.getElementById("download").addEventListener("click", async () => {
-    // Get all data from local storage.
-    const data = await browser.storage.local.get(null);
-    console.debug("Converting JSON to CSV:", data);
-
-    // Extract all object keys to use as CSV headers.
-    const headerSet = new Set();
-    for (const [key, val] of Object.entries(data)) {
-        // Ignore bookeeping information.
-        if (!["initialized", "state"].includes(key)) {
-            for (const [header] of Object.entries(val)) {
-                headerSet.add(header);
-            }
-        }
-    }
-    const headers = Array.from(headerSet);
-
-    let csvData = "";
-
-    // Print one line with each header.
-    for (const [i, header] of headers.entries()) {
-        csvData += `${header}`;
-        if (i == headers.length - 1) {
-            csvData += `\n`;
-        } else {
-            csvData += `,`;
-        }
-    }
-
-    // Print the value for eachs measurement, in the same order as the headers on the first line.
-    for (const [key, val] of Object.entries(data)) {
-        // Ignore bookeeping information.
-        if (!["initialized", "state"].includes(key)) {
-            for (const [i, header] of headers.entries()) {
-                csvData += `${val[header]}`;
-                if (i == headers.length - 1) {
-                    csvData += `\n`;
-                } else {
-                    csvData += `,`;
-                }
-            }
-        }
-    }
+    const db = new Dexie("example");
```

Comment:
Would be good to have a way to clear the DB as well, if someone wanted to have separate and independent test files.

Reply:
Oh good idea, maybe the db name should be configurable at runtime?

Reply:
I think since the db name doesn't show up anywhere in the downloaded file, IMO it would suffice to just have a "Reset Database" button that calls `Dixie.delete("example")`.

Separately but slightly relatedly, it would be nice to have an indicator of how many pings are currently recorded in the db; but I don't want to feature creep this PR!

Reply:
I was planning on updating the UI in a separate PR though, I think we need to decide what framework and get it building etc. which could get large.

Reply:
How about something like this 
<img width="608" alt="image" src="https://user-images.githubusercontent.com/61412/174423647-ec54ed1b-7e1c-43a1-a6d1-ad13bd449ceb.png">

Reply:
I think "Reset Database" or "Clear Database" wording might be more apt, just in terms of the intent